demisto_sdk/commands/split/ymlsplitter.py

```diff
@@ -367,7 +369,18 @@ def print_logs(self, log_msg: str, log_color: str) -> None:
         if self.logging:
             print_color(log_msg, log_color)
 
-    def replace_imported_code(self, script):
+    def split_api_module_for_contribution(self, lines: list, imported_line: str):
+        imported_line_arr = imported_line.split(' ')  # example: imported_line = from CorIRApiModule import *
+        updated_lines = lines[4: len(lines) - 3]  # ignore first 3 lines and last line.
```

Comment:
The comment doesn't match the code. Which is the correct one?
```suggestion
        updated_lines = lines[4:-3]  # ignore first 3 lines and last line.
```